From 2f9a9f008e345ce475f33c278afdad51abb54aee Mon Sep 17 00:00:00 2001
From: Yaroslav Halchenko <debian@onerussian.com>
Date: Tue, 23 Jun 2020 21:43:26 -0400
Subject: [PATCH 2/2] BF(TST): skip one case of
 test_ria_basics.test_binary_data if no network is available

---
 datalad/distributed/tests/test_ria_basics.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/datalad/distributed/tests/test_ria_basics.py b/datalad/distributed/tests/test_ria_basics.py
index fd21b36f9..0328c8200 100644
--- a/datalad/distributed/tests/test_ria_basics.py
+++ b/datalad/distributed/tests/test_ria_basics.py
@@ -22,9 +22,10 @@ from datalad.tests.utils import (
     assert_status,
     assert_true,
     has_symlink_capability,
+    SkipTest,
     known_failure_windows,
+    skip_if_no_network,
     skip_ssh,
-    SkipTest,
     slow,
     swallow_logs,
     turtle,
@@ -497,4 +498,4 @@ def _test_binary_data(host, store, dspath):
 def test_binary_data():
     # TODO: Skipped due to gh-4436
     yield known_failure_windows(skip_ssh(_test_binary_data)), 'datalad-test'
-    yield _test_binary_data, None
+    yield skip_if_no_network(_test_binary_data), None
-- 
2.24.0

